tests/integration/main.rs

```diff
@@ -11,7 +11,7 @@ mod integration_tests {
 
     use crate::network_antenna_reqwest::new_gateway_client;
 
-    const MAX: Duration = Duration::from_secs(3);
+    const MAX: Duration = Duration::from_secs(5);
```

Comment:
I had a bit slow internet connection, better to allow slightly longer time to run integration tests than fail after already have successfully ran doc tests and unit tests. We can also change to START by running integration tests if we want... something we can have in mind.